admin/src/main/java/edu/unc/lib/dl/admin/controller/ExportController.java

```diff
@@ -50,6 +50,9 @@
 @RequestMapping("export")
 public class ExportController extends AbstractSolrSearchController {
 	
+	private static final String DESCRIBED = "Described";
```

Comment:
My only suggestion would be to use the constants that already exist over here https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/master/solr-search/src/main/java/edu/unc/lib/dl/search/solr/util/FacetConstants.java#L20 since this is referencing a specific facet value that is generated using these constants